.github/workflows/cucumber-ruby.yml

```diff
@@ -10,6 +10,9 @@ name: Ruby
 on:
   push:
     branches: [main, publish-option]
+  pull_request:
```

Comment:
I am stuck trying to add an acceptance test which reproduce the issue

This is because the tested code is also the testing code. Because of that we cannot have acceptance tests based on the execution order of the filters in the runtime because it is actually executed a first time before executing the test itself.

At the moment I have not been able yet to write a spec instead.